### ISSUE TITLE
Contact Form inputs are missing accessibility requirements

### ISSUE THREAD

**Issue by robere2** [robere2/Portfolio#20]
Contact Form inputs are missing:
* labels
* placeholders

**Comment by robere2** [comment]
Placeholders will not be added, since I think Chrome only complains if both are missing. As long as one is present, it does not care.
